examples/opf/experiments/opfrunexperiment_test/checkpoints/UNDER_DEVELOPMENT

```diff
@@ -1,423 +0,0 @@
-            # None=use default
-            # Replaces: tpActivationThreshold
-            'activationThreshold': 16,
-
-            'outputType': 'normal',
-
-            # "Pay Attention Mode" length. This tells the TP how many new
-            # elements to append to the end of a learned sequence at a time.
-            # Smaller values are better for datasets with short sequences,
-            # higher values are better for datasets with long sequences.
-            'pamLength': 1,
-        },
-
-        'clParams': {
-            'regionName' : 'SDRClassifierRegion',
-
-            # Classifier diagnostic output verbosity control;
-            # 0: silent; [1..6]: increasing levels of verbosity
-            'verbosity' : 0,
-
-            # This controls how fast the classifier learns/forgets. Higher values
-            # make it adapt faster and forget older patterns faster.
-            'alpha': 0.001,
-
-            # This is set after the call to updateConfigFromSubConfig and is
-            # computed from the aggregationInfo and predictAheadTime.
-            'steps': '1',
-
-
-        },
-
-        'trainSPNetOnlyIfRequested': False,
-    },
-
-
-}
-# end of config dictionary
-
-
-# Adjust base config dictionary for any modifications if imported from a
-# sub-experiment
-updateConfigFromSubConfig(config)
-
-
-# Compute predictionSteps based on the predictAheadTime and the aggregation
-# period, which may be permuted over.
-if config['predictAheadTime'] is not None:
-  predictionSteps = int(round(aggregationDivide(
-      config['predictAheadTime'], config['aggregationInfo'])))
-  assert (predictionSteps >= 1)
-  config['modelParams']['clParams']['steps'] = str(predictionSteps)
-
-
-# Adjust config by applying ValueGetterBase-derived
-# futures. NOTE: this MUST be called after updateConfigFromSubConfig() in order
-# to support value-getter-based substitutions from the sub-experiment (if any)
-applyValueGettersToContainer(config)
-
-
-
-# [optional] A sequence of one or more tasks that describe what to do with the
-# model. Each task consists of a task label, an input spec., iteration count,
-# and a task-control spec per opfTaskSchema.json
-#
-# NOTE: The tasks are intended for OPF clients that make use of OPFTaskDriver.
-#       Clients that interact with OPFExperiment directly do not make use of
-#       the tasks specification.
-#
-control = dict(
-  environment='opfExperiment',
-
-tasks = [
-  {
-    # Task label; this label string may be used for diagnostic logging and for
-    # constructing filenames or directory pathnames for task-specific files, etc.
-    'taskLabel' : "Anomaly",
-
-    # Input stream specification per py/nupic/cluster/database/StreamDef.json.
-    #
-    'dataset' : {
-      'info': 'test_NoProviders',
-      'version': 1,
-
-      'streams': [
-        {
-          'columns': ['*'],
-          'info': 'my simple dataset',
-          'source': 'file://'+os.path.join(os.path.dirname(__file__), 'data.csv'),
```

Comment:
Somehow this file was affecting how the data files were loading in the `description.py` files below.